src/patternfly/components/Card/card.scss

```diff
@@ -115,6 +122,10 @@
     border: var(--pf-c-card--m-flat--BorderWidth) solid var(--pf-c-card--m-flat--BorderColor);
   }
 
+  &.pf-m-no-box-shadow {
```

Comment:
Should this be `pf-m-plain`? As an example, I wonder if we might add another variation that has different padding and a border, and you want to nest that one and remove its border.

Reply:
If it's `pf-m-plain`, I wonder if we should remove the background, too? Do you think the background should be removed from a (sometimes nested) card with no border/shadow?